### ISSUE TITLE
Dropdown options announcing repeatedly and in in-between on JAWS

### ISSUE THREAD

**Issue by dhashmi-madmax** [carbon-design-system/carbon#6986]
## Title line template: [Title]: Brief description

In react application, I have used carbon-component-react Dropdown component. And while AVT3 screen reading testing, dropdown options was reading multiple times and in-between while reading other content on page.

## Environment

> Operating system
    Windows 10 pro
> Browser

> Automated testing tool and ruleset
    JAWS screen reader
> Assistive technology used to verify
    ReactJs
## Detailed description
    Screenshot uploaded below, the selected option reading multiple times on page.
> What version of the Carbon Design System are you using?
   7.11.1
> What did you expect to happen?
   - Dropdown optins should read once.
   - And it should stop reading in-between reading the selected option while reading other text.
> What happened instead?
   NA
> What WCAG 2.1 checkpoint does the issue violate?
  
## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
               <Dropdown
                    ariaLabel={t('XXX')}
                    className={classes.dropdown}
                    disabled={version.length === 1}
                    id='version'
                    label={t('YYY')}
                    items={version}
                    itemToElement={test}
                    initialSelectedItem={version[0]}
                />
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
 
![dropdown avt](https://user-images.githubusercontent.com/72486229/95293270-bc284280-0890-11eb-9945-28b45dbc1e5b.png)

- Notes

**Comment by sstrubberg** [comment]
Let me know if this is still applicable. Happy to re-open.
